modules/subprocess/src/main/scala/notebook/kernel/pfork/BetterFork.scala

```diff
@@ -170,7 +169,7 @@ object BetterFork {
         } else {
           acc ++ (cl.asInstanceOf[URLClassLoader].getURLs map { u =>
             val f = new File(u.getFile)
-            URLDecoder.decode(f.getAbsolutePath, "UTF8")
+            URLDecoder.decode(f.getParent, "UTF8")
```

Comment:
Hey @cbvoxel , I'm sorry I'm completely overwhelmed atm with the Pipeline training at Scala By The Bay next week (http://bigdatascala.bythebay.io/pipeline.html), however, I should be able to look at the consequences late the coming week and get back to you directly then.

sorry man, andy